contentful/sync_page.py

```diff
@@ -38,7 +41,9 @@ def next(self, client):
         return client.sync({'sync_token': self.next_sync_token})
 
     def _get_sync_token(self):
-        return sub(r'.*sync_token=', '', self.next_sync_url)
```

Comment:
There is no need to replace this with `urlsplit` and `parse_qs`, changing `self.next_sync_url` to `self.next_sync_url or self.next_page_url` would accomplish the same result.

Please remove the imports.

Other than that, looks great!